README.md

```diff
@@ -64,17 +64,8 @@ Before starting with the installation it is necessary to install or upgrade the
    cd infraestructure
    ```
 
-4. Navigate to a directory that matches your processor
-
- > [!NOTE]
- > The following command is an example for MacOS and Linux ARM processors. If you are using a different processor, navigate to the corresponding directory.
-
-   ```bash
-   cd MacOS-LinuxARM
-   ```
-
-5. Run the setup script:
+4. Run the setup script:
 
    ```bash
-   ./setup_infrastructure.sh
+   ./setup.sh
```

Comment:
Given this repository is already called "infrastructure", I think it's implied that what we're setting up is the infrastructure, hence setup is enough.

Reply:
Thank!